apps/web/Dockerfile

```
@@ -1,20 +1,44 @@
 WORKDIR /app
-COPY --from=base /app/node_modules ./node_modules
-COPY --from=base /app/packages ./packages
-COPY --from=base /app/apps/web ./apps/web
+COPY package.json ./package.json
+COPY packages/ ./packages/
+COPY apps/web/package.json ./apps/web/package.json
+RUN --mount=type=cache,target=/root/.bun \
+    bun install --no-cache
+
+FROM node:22-alpine AS runtime
+RUN apk add --no-cache \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)